pkg/decode/bytes.go

```diff
@@ -63,6 +64,10 @@ func (b *BytesReader) Int8() (v int, err error) {
 	return int(uv), err
 }
 
+func (b *BytesReader) IntEndIdx() (v int) {
+	return bytes.IndexByte(b.data[b.off:], byte(0))
```

Comment:
Should we consider returning an error if we can't find a byte(0)?

Reply:
done